### DISCUSSION TITLE
데이터 API를 불러와서 통계나 정보를 보여주는 앱

### DISCUSSION THREAD

**Discussion by ypd01018** [ondal-house/zu-story#7] [아이디어 선정 안건]
(시각화 Grafana 쓸수도잇고)\(Op.GG\

**Comment by ResinCRT** [comment]
데이터 수집의 경우 통계 데이터 API를 제공해주는 곳(보통 게임 데이터가 그럼)> 크롤링 쉬운거 > 암것도 아님 순으로 난이도가 낮음

- 시각화 여부는 통계 데이터에 따라서? 
- 데이터 검색 기능

단점}
페이지나 기능이 적으니 분석이나 시각화나 다른 기능을 생각해봐야 할 것 같음
